test/e2e_node/system/docker_validator_test.go

```diff
@@ -28,7 +28,7 @@ func TestValidateDockerInfo(t *testing.T) {
 		Reporter: DefaultReporter,
 	}
 	spec := &DockerSpec{
-		Version:     []string{`1\.(9|\d{2,})\..*`},
+		Version:     []string{`1\.(9|\d?[0-2]{2,})\..*`}, // Requires 1.9+
```

Comment:
sig-node agreed to drop 1.9.x support for 1.6 release, can we update the test here reflecting such change too?